syntaxes/samarium.tmLanguage.json

```diff
@@ -111,22 +91,42 @@
                     }
                 },
                 {
-                    "match": "([A-Za-z0-9]+|=>)((?:\\s*[A-Za-z0-9]+\\s*(?:\\?|\\.{3})?)*)\\s*(\\*)\\s*",
+                    "match": "([A-Za-z0-9_]+|=>)((?:\\s*[A-Za-z0-9_]+\\s*(?:\\?|\\.{3})?)*)\\s*(\\*)\\s*",
```

Comment:
Just change `[A-Za-z0-9_]` to `\w` at this point

Reply:
true lmao

Reply:
should i do it for classes too?

Reply:
everywhere ig